File-Formats/DocIO/FAQ.md

```diff
@@ -1912,127 +1063,65 @@ Yes, you can specify the PDF conformance level in Word to PDF conversion. For mo
 
 ### Mail merge
 
-The Mail merge feature can be used to generate reports and letters in Microsoft Word. The following code examples show how to generate an employee report from an MDB data source by using Office Automation and DocIO.
-
-Using Microsoft Office Automation
+The Mail merge feature can be used to generate reports and letters in Microsoft Word. The following code examples show how to generate an employee report from an MDB data source by using Microsoft Office Automation and DocIO.
 
```

Comment:
You have removed this heading line "Using Microsoft Office Automation"

No need to remove it.